src/applications/mhv-medications/components/RefillPrescriptions/RenewablePrescriptions.jsx

```diff
@@ -73,21 +73,25 @@ const RenewablePrescriptions = ({ renewablePrescriptionsList = [] }) => {
           data-testid="medications-page-link"
           to="/"
           data-dd-action-name={`Go To Your Medications List Action Link - ${
-            DD_ACTIONS_PAGE_TYPE.REFILL
-          } - Renew Section`}
+            DD_ACTIONS_PAGE_TYPE.LIST
```

Comment:
Did someone ask to change this? I don't see it in the AC's of the ticket. I only ask because I gave a list to Anne of exact datadog action names so she might be expecting to see it the old way

Reply:
@robertbylight did

Reply:
Removed